packages/java-packages/snippet-replacer-maven-plugin/src/main/java/azuresdk/plugin/SnippetDictionary.java

```diff
@@ -0,0 +1,38 @@
+package azuresdk.plugin;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class SnippetDictionary {
+    private HashMap<String, List<String>> _snippetDictionary = new HashMap<String, List<String>>();
+
+    public SnippetDictionary(){
+
+    }
+
+    public boolean isActive(){
+        return !_snippetDictionary.isEmpty();
+    }
+
+    public void beginSnippet(String key) {
+        if(!this._snippetDictionary.containsKey((key)))
+        {
```

Comment:
nit: opening braces on same line, rather than a new line.

Reply:
I'll do a pass across the whole thing and fix all these.